system-variables.md

```diff
@@ -1711,6 +1712,7 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
 - Range: `[1, 256]`
 - Unit: Threads
 - This variable is used to set the concurrency of the DDL operation in the `re-organize` phase.
+      - Starting from version v8.3.0, this variable supports session-level settings. As a result, modifying the global-level variable value will not affect DDL jobs currently in progress but will only take effect for DDL statements submitted in newly created sessions.
```

Comment:
```suggestion
- Starting from v8.3.0, this parameter is supported at the SESSION level. Modifying the parameter at the GLOBAL level will not impact currently running DDL statements. It will only apply to DDLs submitted in new sessions.
```